tests/conftest.py

```diff
@@ -25,6 +25,7 @@
 
 from deepchecks.tabular import Dataset
 
+from .vision import vision_conftest
```

Comment:
won't this load all vision stuff when testing the general package? Do we want that?